src/api.ts

```diff
@@ -181,7 +181,7 @@ export const serviceUsageOrigin = utils.envOverride(
 );
 export const frameworksOrigin = utils.envOverride(
   "FRAMEWORKS_URL",
-  "https://placeholder.googleapis.com"
+  `${process.env.FRAMEWORKS_URL}`
```

Comment:
You don't need to do this. It is automatically handled by `utils.envOverride` utility being used here.

You can just leave this code alone...